docs/dev_guide/new_objects.rst

```diff
@@ -4,43 +4,88 @@
 Creating new SunPy Subclasses (Maps, TimeSeries)
 ************************************************
 
+Documenting new data sources
```

Comment:
I don't think this should go above the instructions for how to make the class, it should go below.

Reply:
I think it makes sense to have it in here. I would suppose it's more likely people will see it if it's here.